spring-ai-core/src/main/java/org/springframework/ai/vectorstore/impl/InMemoryVectorStore.java

```diff
@@ -95,6 +95,14 @@ public Similarity(String key, double similarity) {
 			this.similarity = similarity;
 		}
 
+		public double getSimilarity() {
```

Comment:
Not sure those methods in the InMemoryVectorStore are relevant to the MongoDB store?